### DISCUSSION TITLE
Add a link to the lexicon wiki in the plugin settings?

### DISCUSSION THREAD

**Discussion by johnysandels** [karashiiro/TextToTalk#76] [Ideas]
Seems like people don't know there is already lexicon files made that they can download. So it could be nice to have a button to click that would open up the lexicon wiki page in a browser.
Thoughts?

**Comment by karashiiro** [comment]
I think this is a good idea, I can add something near the current lexicon settings 👍

**Comment by karashiiro** [comment]
Done 👍 
https://github.com/goatcorp/DalamudPlugins/pull/1977
https://github.com/karashiiro/TextToTalk/commit/4c2449d1bda7ee7ef39e0c493c35059e22ae9219
